runtests.sh

```diff
@@ -101,6 +115,22 @@ function compile_cpp {
     fi
 }
 
+function clang_format {
+    echo "Running clang-format..."
+    ${cmdPrefix}python -m tests.clang_format_utils
+    clang_format_tool='.clang-format-bin/clang-format'
+    # Verify .
+    if ! type -p "$clang_format_tool" >/dev/null; then
+        echo "'clang-format' not found, skipping the formatting."
+        exit 1
+    fi
+    if ! type -p "git" >/dev/null; then
+        echo "'git' not found, skipping the formatting."
+        exit 1
+    fi
+    git ls-files | grep -e "monai/csrc" | tr '\n' '\0' | xargs -0 "$clang_format_tool" -style=file -i
```

Comment:
Replacing newlines with nulls actually works with clang-format as inputs? I would think it would complain about being unable to understand those arguments

Reply:
I would suggest this instead (assuming the style file is $style_file):
```sh
find monai/csrc -type f | while read i; do clang-format "--style=$style_file" -i $i; done
```
or:
```sh
find monai/csrc -type f -exec clang-format "--style=$style_file" -i '{}' ';'
```

Reply:
thanks!